### ISSUE TITLE
[tailwindcss-react-aria-components] Cannot be used in Bun >1.0.13 because of mixing import & module.exports

### ISSUE THREAD

**Issue by iRyusa** [adobe/react-spectrum#5579]
### Provide a general summary of the issue here

Bun cannot mix both since 1.0.14 update https://bun.sh/blog/bun-v1.0.14#errors-for-unsupported-esm-commonjs-features

### 🤔 Expected Behavior?

To be able to import/require `tailwindcss-react-aria-components` in a Bun Project

### 😯 Current Behavior

`require('tailwindcss-react-aria-components')` returns an empty object 

### 💁 Possible Solution

Either use `export default` or `require` instead of `import`

### 🖥️ Steps to Reproduce

Having a tailwind config with Bun >=1.0.14
```
/** @type {import('tailwindcss').Config} */

export default {
  plugins: [require('tailwindcss-react-aria-components'), require('tailwindcss-animate'), require('autoprefixer')]
}
```

### Version

Latest

### What browsers are you seeing the problem on?

Other

### If other, please specify.

n/a

### What operating system are you using?

OSX

**Comment by iRyusa** [comment]
Seems to be fixed with #5541 but not yet push to NPM ! 👍

**Comment by reidbarber** [comment]
@iRyusa It should be included in the upcoming release, but is also available now in the nightly build now.

**Comment by iRyusa** [comment]
Yup sorry about that i was about to make a PR and found out it was done when i've forked it 😅
